scripts/delete.sh

```diff
@@ -115,7 +118,17 @@ if [[ ! -z $user_ns_array ]]; then
   rm runtimes.yaml
 fi
 
+# Determine whether a modelmesh-controller-rolebinding clusterrolebinding exists and is
+# associated with the service account in this namespace. If not, don't delete the cluster level RBAC.
+set +e
+crb_ns=$(oc get clusterrolebinding modelmesh-controller-rolebinding -o json | jq -r .subjects[0].namespace)
```

Comment:
Shall we stay with kubectl instead of oc?

Reply:
Thanks @chinhuang007 good catch, now fixed.